e2e/stress/IoTClientPerf/readme.md

```diff
@@ -0,0 +1,58 @@
+# Azure IoT Hub Stress Test
+
+Use `dotnet run` to display the help page for this tool.
```

Comment:
FYI: I got the following error on running dotnet run from cmd:
`Reporting\ResultWriterFile.cs(24,89): error CS1738: Named argument specifications must appear after all fixed arguments have been specified. Please use language version 7.2 or greater to allow non-trailing named arguments.`
It ran after I modified the Language version in VS 2017 to use latest minor version.

Reply:
Thanks! I think I'm on latest and using VS2019 which means it'll work for me w/o the warning.
I'll remove this to allow older .NET versions.